packages/tools/docs-tools/src/utils/getFileExtension.ts

```diff
@@ -0,0 +1,9 @@
+import path from 'path';
+
+export const getFileExtension = (filePath: string): string => {
+  const extensionWithHash = path.extname(filePath).substring(1);
+
+  //also remove the # at the end of a link
+  const extensionArr = extensionWithHash.split('#');
```

Comment:
```suggestion
  const extensionArr = extensionWithHash.split(/[?#]/);
```
this could be better so we don't have to worry about queryParams and hash strings

Reply:
nice catch!!!

Reply:
also added extra test for it